cns/restserver/ipam.go

```diff
@@ -1068,6 +1084,11 @@ func (service *HTTPRestService) GetEndpointHelper(endpointID string) (*EndpointI
 		logger.Warnf("[GetEndpointState] Found existing endpoint state for container %s", endpointID)
 		return endpointInfo, nil
 	}
+	legacyEndpointID := endpointID[:ContainerIDLength] + "-" + req.IFName
```

Comment:
can you add a comment on why this change is required?

Reply:
added